packages/schema/__tests__/util/ascendToPackageDir.ts

```diff
@@ -0,0 +1,17 @@
+import fs from "fs";
+import path from "path";
+
+export function ascendToPackageDir(packageName: string = "breadboard-ai") {
+  let directory = process.cwd();
```

Comment:
Avoid cwd, better to use import.meta.url to make paths relative to the current file. The user controls cwd so it can be anything, but import.meta.url will always be reliable.